include/exadg/structure/postprocessor/output_generator.cpp

```diff
@@ -73,6 +73,18 @@ OutputGenerator<dim, Number>::setup(dealii::DoFHandler<dim> const & dof_handler_
                          mpi_comm);
     }
 
+    // write grid
```

Comment:
this was not there before, but as useful as for the fluid mechanics solver:
usecase:
you want the grid with mapping, apart from that only data that is not on the grid (currently only errors writte to the log file).